src/UIState/SeePIDConstants.h

```diff
@@ -23,7 +23,8 @@ class SeePIDConstants : public UIState {
   void loadKp(uint16_t line);
   void loadKi(uint16_t line);
   void loadKd(uint16_t line);
-  void loadSlope(uint16_t line);
+  void loadPID(uint16_t line);
+//   void loadSlope(uint16_t line);
```

Comment:
If we find we don't need this file (things work with it commented out), let's remove the line.

Reply:
Note also that this line fails the formatting test, so when it is removed we should have passing tests!